src/main/resources/database/changelog-master.xml

```diff
@@ -0,0 +1,7 @@
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <include file="classpath:/database/changelog/changelog-tables.xml"/>
```

Comment:
any changelog might have changes in tables, so it's a wrong name. 
https://habr.com/ru/post/179425/
The article is overcomplicated for microservice, but the main point is that your changelogs should be versioned